.travis.yml

```diff
@@ -34,7 +34,7 @@ matrix:
       cargo check --all
 
   # This runs TSAN against nightly and allows failures to propagate up.
-  - rust: nightly
+  - rust: 1.32.0-nightly
```

Comment:
I thought the syntax for this was something like `nightly-20YY-MM-DD`? Didn't know that was also valid syntax!

Reply:
Might not be valid.. I'm just trying it out :P